proto/babylon/btcstaking/v1/btcstaking.proto

```diff
@@ -109,6 +109,24 @@ message BTCDelegation {
     uint32 params_version = 16;
 }
 
+// DelegatorUnbondingInfo contains the information about transaction which spent
+// the staking output. It contains:
+// - spend_stake_tx: the transaction which spent the staking output
+// - spend_stake_tx_inclusion_block_hash: the block hash of the block in which
+// spend_stake_tx was included
+// - spend_stake_tx_sig_inclusion_index: the index of spend_stake_tx in the block
+message DelegatorUnbondingInfo {
+    // spend_stake_tx is the transaction which spent the staking output. It is
+    // filled only if spend_stake_tx is different than unbonding_tx registered
+    // on the Babylon chain.
+    bytes spend_stake_tx = 1;
+    // spend_stake_tx_inclusion_block_hash is the block hash of the block in which
+    // spend_stake_tx was included
+    bytes spend_stake_tx_inclusion_block_hash = 2;
+    // spend_stake_tx_inclusion_index is the index of spend_stake_tx in the block
+    uint32 spend_stake_tx_inclusion_index = 3;
```

Comment:
Do we need to keep those in the KV store? It seems not as they are only be used upon verification but nothing else

Reply:
tbh my main use for those fields was to make it possible to easily check this transaction on Bitcoin node i.e when you have blockhash and transaction index you can retrieve tx from BTC node even if it does not have `--txindex` flag. 

So later it is easier to check unbondings (normal or not) against BTC network , and the cost is only additional 40bytes of data.  Do you think it is useful or overkill from node perspective ?

Reply:
It's a bit overkill imo -- since we have millions of BTC delegations these extra fields might still incur some cost in the states. Also since we know the tx hash, we can find relevant block that contains this tx using Bitcoind APIs. Wdyt?

Reply:
>It's a bit overkill imo -- since we have millions of BTC delegations these extra fields might still incur some cost in the states

For 5m delegation it will be around 50MB of data which seems negligible, but I understand your point: 50MB here, 50MB there and it adds up.

In https://github.com/babylonlabs-io/babylon/pull/215/commits/74db17e7881cc0580cfa334d347cf5afc66f0d8b:
- I have removed it from the state
- though I left it in the event, as this is not part of the state and this piece of data maybe important for somebody